### ISSUE TITLE
Create an API technical prototype of deleting an org member

### ISSUE THREAD

**Issue by echappen** [cloud-gov/cg-ui#161]
Blocked by #162 

Along the lines of #102, this prototype will test the following scenario:

- Org Manager can remove a member from a CloudFoundry organization

The prototype will verify:

- What API calls are necessary for this scenario?
- What kinds of error messages could be returned?

### Considerations

- This may entail moving the list of org members from server-rendered to client-rendered
- Styling is out of scope (other than what's necessarily functional for the layout/button)

### Acceptance criteria

- [x] From the list of org members, adds a button beside each one to remove that member from the org
- [x] When button is clicked, a confirmation message shows. Confirming removes the member.
- [x] Shows error message on failure / success message on success
- [x] List of members updates with member removed
